refill-style.yaml

```diff
@@ -716,17 +583,20 @@ styles:
         dash: [0.25, 1.0]
         dash_background_color: global.offwhite_color
 
+    region_dash:
```

Comment:
We can eliminate this style now that line `dash` can be assigned in `layers`, within a `draw` group.

Reply:
Actually all of the dashed styles above (`dashed`, `dashed_big`, `dashed_steps`, `railway_dash`, and `region_dash`) can be eliminated, and the desired `dash` or `dash_background_color` values can be set within the appropriate `draw` block, using regular `lines`.

Reply:
I found it, but it's not in the documentation

Reply:
https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#dash

Reply:
yes, found it on github docs, but not on mapzen website docs

Reply:
ok this is done